docs/continuous-integration/dev-sec-ops/evaluate-oss/README.md

```diff
@@ -0,0 +1,29 @@
+# Evaluate Open Source Software
+
+Identify potential candidates for open-source dependencies and evaluate them against your needs and the required security posture.
+
+## Why Evaluate Open Source Software
```

Comment:
I would consider adding this as examples:
https://david-gilbertson.medium.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5

Reply:
added another one (Microsoft one :) )